### PR TITLE
WIKI-1290 Only first two lines of the wiki page are dispplayed in the…

### PR DESCRIPTION
**Pull request by nttuyen**
- Check for empty line when creating excerpt for wiki page creating activity

### CODE REVIEW

File: integ-wiki/integ-wiki-social/src/main/java/org/exoplatform/wiki/ext/impl/WikiSpaceActivityPublisher.java [hunk 1]
```diff
@@ -203,18 +205,24 @@ private ExoSocialActivity generateActivity(Identity ownerStream,
...
+        }      
+    }).limit(4).forEach(line -> {
+      result.append(line);
       result.append("\n");
-    }
-    
+    });
+
     if (lines.length > 4) {
       result.append("...");
```
Comment (boubaker) [inline integ-wiki/integ-wiki-social/src/main/java/org/exoplatform/wiki/ext/impl/WikiSpaceActivityPublisher.java]: if we have 5 lines and an empty line inside those lines, the "..." will be displayed at the end while it shouldn't
(please fix this and update unit test accordingly)

**Comment by nttuyen** [comment]
@boubaker , i've done fix that case, and add unit test

File: integ-wiki/integ-wiki-social/src/main/java/org/exoplatform/wiki/ext/impl/WikiSpaceActivityPublisher.java [hunk 2]
```diff
@@ -202,20 +205,28 @@ private ExoSocialActivity generateActivity(Identity ownerStream,
...
+      .map(new Function<String, String>() {  
+        @Override
+        public String apply(String line) {
+          if (line.length() > EXCERPT_LENGTH) {
+            line = line.substring(0, EXCERPT_LENGTH) + "...";
+          }
+          return line;
+        }      
+    }).limit(4).forEach(line -> {
+      result.append(line);
```
Comment (boubaker) [inline integ-wiki/integ-wiki-social/src/main/java/org/exoplatform/wiki/ext/impl/WikiSpaceActivityPublisher.java]: This works as expected, except that the line is not displayed in a different line on activity.
IMO, you should change "\n" by  <p>LINE</p>

**Comment by nttuyen** [comment]
@boubaker , ok , i'm done using <p> instead of \n

**Pull request merged**